tasks/edit_block/__init__.py

```
@@ -47,69 +51,78 @@ def main(params: Inputs, context: Context) -> Outputs:
+        }
+        
+        response = requests.post(
+            initiate_url, 
+            files=files_array, 
+            data=data, 
+            headers=headers, 
+            timeout=120
+        )
+        response.raise_for_status()
+        
+        # nano-banana/edit 模型是直接返回结果，不需要查询进度
```

Ruff (0.12.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)